bedrock/mozorg/templates/mozorg/plugincheck.html

```diff
@@ -55,6 +55,11 @@
   {# L10n: This is the string used to indicate that the plugin is unknown #}
   data-unknown="{{ _('unknown') }}"
 
+  {% if l10n_has_tag('plugin_outdated') -%}
+    {# L10n: This is the string used to indicate that the plugin is outdated #}
+    data-outdated="{{ _('outdated') }}"
+  {% endif %}
```

Comment:
I'm not sure what the best approach is here, but the translation needs to fall back to something. Agree it seems confusing :/